docs/sources/shared/datasources/tempo-editor-traceql.md

```diff
@@ -58,35 +58,37 @@ To query a particular trace by its trace ID:
 1. Enter the trace ID into the query field. For example: `41928b92edf1cdbe0ba6594baee5ae9`
 1. Click **Run query** or use the keyboard shortcut Shift + Enter.
 
-![Search for a trace ID using the TraceQL query editor](/static/img/docs/tempo/screenshot-traceql-editor-traceID.png)
+![Search for a trace ID using the TraceQL query editor](/media/docs/tempo/traceql/screenshot-traceql-editor-traceID.png)
```

Comment:
This line has a conflict in the PR. If you try to resolve the conflicts, use the content from the version in this PR.